flutter-for-react-native.md

```diff
@@ -1707,7 +1707,8 @@ void _submit() {
 
 ## Platform-specific code
 
-When building a cross-platform app, you'll want to re-use as much code as possible across platforms. Scenarios may arise where it makes sense for the code to be different. In React Native, separate implementation can be given at almost anywhere in the app code by recognizing the platform the app will run on. To get the target platform, you need to use this component :
+When building a cross-platform app, you'll want to re-use as much code as possible across platforms. Scenarios may arise where it makes sense for the code to be different. In React Native, separate implementation can be given at almost anywhere in the app code by recognizing the platform the app will run on. To get the target platform,
+you can either use platform file extensions or the `Platform` module as below:
```

Comment:
In React Native, separate... => In React Native, a separate...
either use platform file extensions or => either use platform file extensions, or

Another problem I'm noticing. Lines of text should never be more than 80 chars long. You don't have to fix pre-existing lines, but please make all new lines 80 chars.

Reply:
Would it be worth adding prettier to the repo to automate this away? Happy to raise a PR.